netcode.c

```diff
@@ -3794,18 +3794,36 @@ void skillz_print_all_matches( struct netcode_server_t * server )
 }
 
 /**
+ * @brief skillz_match_delete
```

Comment:
Is this trying to say that free == delete?  Shouldn't this be a brief one-liner?

Reply:
Changed the function name, forgot to change the comment.